### ISSUE TITLE
Hello! 🤍 simple question

### ISSUE THREAD

**Issue by 4lt3r3go** [Tencent/HunyuanVideo#125]
A question regarding the 256p pool mentioned in the pdf here https://arxiv.org/pdf/2412.03603
![image](https://github.com/user-attachments/assets/70f3ec92-b2d0-4b9e-b17f-f6cabdd44dae)

. Could you please clarify what exact resolution (width x height) this 240p corresponds to? 
what specific resolution should I aim for if I want to target that specific 256p pool to get more variety in my outputs?
especially considering different aspect ratios like 16:9 or 4:3? 

Thank you in advance for your help🤍
sincerly, a big fan

**Comment by OrphBean** [comment]
Yes this is a great question i am keen on this answer too as i am looking for diversity of output

**Comment by JacobKong** [comment]
Thanks for your attention. You need to keep hxw=256x256 while changing the height and width.

**Comment by 4lt3r3go** [comment]
🤍thaks to you! 
do you mean the total amout of pixel must be 256x256
OR
keep one side 256 and change the other, like to match a bucket?

**Comment by zappazack** [comment]
> 🤍thaks to you! do you mean the total amout of pixel must be 256x256 OR keep one side 256 and change the other, like to match a bucket?

I was curious because I recognised the same variety of outputs changing the resolution. I have asked chatgpt about the 256x256 here is the answer: 
The answer is essentially saying that if you're targeting a specific 256p pool (which typically refers to a low-resolution format), you should keep the total number of pixels constant at 256x256, but you can adjust the width and height to fit different aspect ratios (like 16:9, 4:3, or vertical videos).

Here’s what this means in more detail:

- The total pixel count (height × width) should remain at 256 × 256 = 65,536 pixels.
- You can change the width and height while maintaining that pixel count. For example:
  - For a 16:9 aspect ratio, you could set the width to 256 pixels, and the height would be around 144 pixels (since 256 × 144 = 65,536 pixels).
  - For a 4:3 aspect ratio, you could set the width to 256 pixels, and the height would be 192 pixels (since 256 × 192 = 65,536 pixels).
  - For vertical videos, you could use a height of 256 pixels and adjust the width accordingly.

The goal is to adjust the width and height in such a way that the total number of pixels remains 65,536, while allowing for different aspect ratios in your output.

**Comment by zappazack** [comment]
> 🤍thaks to you! do you mean the total amout of pixel must be 256x256 OR keep one side 256 and change the other, like to match a bucket?

Same issue here very high variety of outputs caused by different resolutions. I have asked Chatgpt about the resolution 256x256 =65536px not sure of the answer is 100% correct as the maths was not properly calculated. Last answer seems fine to me.

![Image](https://github.com/user-attachments/assets/e4f6771b-73ac-4181-963f-88ca24173f52)

**Comment by 4lt3r3go** [comment]
all this is still a mistery in february 🤣
